apps/settings/js/panels/root/panel.js

```diff
@@ -158,3 +133,29 @@ define(function(require) {
     });
   };
 });
+/**
+ * Update the sim and NFC related items
+   based on mozMobileConnections and Device.
+ */
+
+var updateSimItemsAndNfc = function rph_refrehsSimItems() {
```

Comment:
1. other code for Network and Connectivity is in root/panel.js because in mozilla's version all functionality located in 'root' panel, you are moving UI for this functionality to separate panel, so from architectural point of view, you shall move all related logic also, since in your implementation this functionality isn't required for the root panel anymore. You are changing structure of application, so, to be consistent, model and view shall be both updated, not only view part.
 2. ok, since you are moving NFC item to the 'network' panel, it could remain in same function as update of other network related items (though they are not logically bound) but anyway this code needs to be moved to new panel's module, e.g. panels/network/panel.js as mentioned in previous point.
 3. as I can see this item was deleted from elements/root.html in commit f00a0491c775160 made by inagupta020 <Ankit.Gupta020@Symphonyteleca.com>  2015-10-26 15:24:12, but wasn't added to any other .html file. So, just for example, your code ``document.getElementById('simCardManager-settings').hidden = true;`` will most probably throw an exception, since you are trying to access ``hidden`` property of nothing, thus by the way your NFC-related code won't be executed at all.
 4. exactly, that's why code related to particular relocated items shall be moved to proper panel(s) instead of being executed in 'root' panel context.